--- .upstream/outputrevision.h	2022-06-30 14:11:55.907629780 +0200
+++ outputrevision.h	2022-06-17 10:43:13.000000000 +0200
@@ -4,7 +4,13 @@
 #ifndef OUTPUTREVISION_H
 #define OUTPUTREVISION_H
 
+#include <QtCore/qglobal.h> // -- QtScxml
+
+QT_BEGIN_NAMESPACE // -- QtScxml
+
 // if the output revision changes, you MUST change it in qobjectdefs.h too
 enum { mocOutputRevision = 68 };          // moc format output revision
 
+QT_END_NAMESPACE // -- QtScxml
+
 #endif // OUTPUTREVISION_H
